tests/integration/distribution.go

```diff
@@ -554,22 +553,23 @@ func (s *CCVTestSuite) TestIBCTransferMiddleware() {
 		},
 		{
 			"IBC Transfer recipient is not the consumer rewards pool address",
-			func(ctx sdk.Context, keeper *providerkeeper.Keeper, bankKeeper icstestingutils.TestBankKeeper) {
-				data.Receiver = "cosmos149lw9fktlqfed3zt8ah48r5czmsug5s7kw77u9" // random acct address
+			func(s *CCVTestSuite, ctx sdk.Context, keeper *providerkeeper.Keeper, bankKeeper icstestingutils.TestBankKeeper) {
+				data.Receiver = s.providerChain.SenderAccount.GetAddress().String() // random acct address
```

Comment:
This is not a random address. It seems that it will always be the same address. Can't we use a variable declared before `testCases` and avoid adding a new param to the setup function?

Reply:
The comment was misleading, I adjusted it.

I don't understand the concrete proposal. In all testcases except this one, it is important to keep the address as-is

Reply:
Also adjusted the logic to not need the testsuite here.